### ISSUE TITLE
ControllerStorage was already loaded!

### ISSUE THREAD

**Issue by marcus-j-davies** [homebridge/HAP-NodeJS#895]
Hello,

I have a need (or a feature) to allow users to take their Bridge off-line (and naturally putting it back online).
I do this via `publish` (to go on-line)
and `unpublish` (to go off-line)

Users can create new accessories (attached or unattached to a bridge), and I want the ability for users to be able to take down (and put back online) the bridge

However, after calling `publish` on the bridge class (after it was once unpublished)

I get

```
ControllerStorage was already loaded!
at ControllerStorage.load (/hap-nodejs/src/lib/model/ControllerStorage.ts:212:19)
at Bridge.Accessory.publish (/hap-nodejs/src/lib/Accessory.ts:1167:28)
```

Is this not possible? (I do not destroy the accessory - I am only calling `unpublish`)
I should also mention this unpublish/publish can happen on the same process run

Thanks

**Comment by marcus-j-davies** [comment]
As this has been tagged differently  - this is using 0.9.4

**Comment by marcus-j-davies** [comment]
@Supereg,

Just wondering if this should be possible, given the change to `BUG`.
if `unpublish`/`publish` in the same process - is not possible I'll source an alternative method

**Comment by Supereg** [comment]
Issue was fixed with the latest v0.9.7 release. 

Please not, that calling `publish` on an accessory object which was `unpublish`ed isn't a well tested use case. With the reference fix I tried to cover this better by our test suite.
Please report any further issue if you discover some.

Thanks for your contribution.
